ApplyToTheIndustry/Assets/Scripts/UI/OnboardingManager.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+public class OnboardingManager : MonoBehaviour
+{
+    // Public fields
+    public GameObject[] buttons;
```

Comment:
In the same vein, maybe we want to create Tutorial objects that this onboarding manager can hold a list of. That way it can keep track of every tutorial separately, reducing coupling a bit while at the same time allowing us to create new tutorials a bit faster.